### ISSUE TITLE
I don't think this code does what you think

### ISSUE THREAD

**Issue by ghost** [d3v1l401/Open-Xign-Crypto#1]
I've recently taken to reversing Xigncode, and happened to stumble across this repository. 
I realize this repo is pretty old, and by posting this I'm not trying to be mean or disparaging, but rather to prevent the confusion of other people who may also find this repo. That said, I feel like I should clear a couple things up.

I don't think the functions you reversed do what you think they do, specifically the `Decrypt`, `CreateSBox`, and `DecryptWrap` functions in `OXC Manager.cpp`.

In fact, `DecryptWrap` is literally just the MSVC implementation of `memcpy` (look at `c:\windows\syswow64\msvcrt.dll!memcpy` in IDA, it's almost the exact same disassembly depending on the version). Your `UseLowLevelDecryption` dword is the `__sse2_available` variable that determines if your CPU supports faster copies with sse2 instructions. `Decrypt` is actually the `_VEC_memcpy` function, and `CreateSBox` is an unnamed function used by `_VEC_memcpy` that implements 128-bit copies using XMM registers.

In other words, unfortunately nothing here really has anything to do with Xigncode or even encryption at all.

**Comment by d3v1l401** [comment]
This is extremely old and wasn't supposed to be a full Crypto api emulation, but a simple dump.
